includes/admin/metaboxes/settings.php

```diff
@@ -1109,6 +1128,10 @@ public static function save($post_id, $post)
 		 * Calendar settings panel *
 		 * ======================= */
 
+		// Print calendar.
+		$static = isset($_POST['_display_print_calendar']) ? 'yes' : 'no';
```

Comment:
Can we instead save boolean?

Reply:
the "simcal_print_field" function, which manage the field in admin, accepts the value in yes/no format, that why I have use the same.